modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/TableDescriptor.java

```diff
@@ -41,6 +41,9 @@ public interface TableDescriptor extends InitializerExpressionFactory, Iterable<
      */
     RelDataType rowType(IgniteTypeFactory factory, @Nullable ImmutableBitSet usedColumns);
 
+    /** Returns type of the row excluding all {@link ColumnDescriptor#hidden() hidden} columns. */
+    RelDataType rowTypeSansHidden();
```

Comment:
Why `Sans`? What does it mean?

Reply:
this means "without" or "ignoring" ([link](https://dictionary.cambridge.org/dictionary/english/sans) to dictionary). This term is used in core calcite in a few places:
* `SqlTypeUtil#equalSansNullability(org.apache.calcite.rel.type.RelDataType, org.apache.calcite.rel.type.RelDataType)`
* `RelTraitSet#isDefaultSansConvention`
* `RelDataType#equalsSansFieldNames`
* `SqlValidatorNamespace#getRowTypeSansSystemColumns`